include/libkahypar.h

```diff
@@ -99,8 +99,6 @@ KAHYPAR_API void kahypar_improve_hypergraph_partition(kahypar_hypergraph_t* kahy
                                                       kahypar_partition_id_t* improved_partition);
 
 
-KAHYPAR_API void kahypar_hypergraph_free(kahypar_hypergraph_t* kahypar_hypergraph);
```

Comment:
Why do we need to remove this?

Reply:
This line appeared twice in the file, which caused a compiler warning for me. Thus I removed one of the duplicates